### PR TITLE
Rwa 80 add standard attributes

### PR DESCRIPTION
**Pull request by chrisgrimble**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/wa-task-configuration-api/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for wa-task-configuration-api App
 name: wa-task-configuration-api
 home: https://github.com/hmcts/wa-task-configuration-api
-version: 0.0.2
```
Comment (alexbaeza) [inline charts/wa-task-configuration-api/Chart.yaml:6]: need to bump down to 0.0.3

File: charts/wa-task-configuration-api/values.yaml [hunk 2]
```diff
@@ -3,9 +3,19 @@ java:
   image: 'hmctspublic.azurecr.io/wa/task-configuration-api:latest'
   ingressHost: wa-task-configuration-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: wa
+  environment:
+    CAMUNDA_URL: "http://camunda-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/engine-rest"
+    IDAM_URL: "https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net"
+    S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    CCD_URL: "http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    FORCE_REBUILD: 1
```
Comment (alexbaeza) [inline charts/wa-task-configuration-api/values.yaml:11]: is the FORCE_REBUILD needed ?

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/ConfigureTaskTest.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+import static net.serenitybdd.rest.SerenityRest.given;
+import static org.hamcrest.CoreMatchers.is;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.reform.wataskconfigurationapi.CreateTaskMessageBuilder.createBasicMessageForTask;
+import static uk.gov.hmcts.reform.wataskconfigurationapi.CreatorObjectMapper.asCamundaJsonString;
+import static uk.gov.hmcts.reform.wataskconfigurationapi.CreatorObjectMapper.asJsonString;
+
+@RunWith(SpringIntegrationSerenityRunner.class)
+@SpringBootTest
+@ActiveProfiles("functional")
```
Comment (alexbaeza) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/ConfigureTaskTest.java:35]: perhaps we can move this to a base test and extend from it

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/ConfigureTaskTest.java [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+public class ConfigureTaskTest {
+    public static final DateTimeFormatter CAMUNDA_DATA_TIME_FORMATTER = ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+    private final String camundaUrl =
+        System.getenv("CAMUNDA_URL") == null ? "http://localhost:8080/engine-rest" : System.getenv("CAMUNDA_URL");
+    private final String testUrl =
+        System.getenv("TEST_URL") == null ? "http://localhost:8091" :  System.getenv("TEST_URL");
```
Comment (alexbaeza) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/ConfigureTaskTest.java:41]: we could probably use the `@ Value` here

File: charts/wa-task-configuration-api/values.yaml (hunk 2 above)
Comment (chrisgrimble) [reply charts/wa-task-configuration-api/values.yaml:11]: I know case-api have something like that but after yesterdays talk with rpe don't think we need it.

**Review by alexbaeza — DISMISSED** [review]

**Review by alexbaeza — APPROVED** [review]

**Pull request merged**